src/components/collection/copyModal/RootNavigationTree.tsx

```diff
@@ -0,0 +1,87 @@
+import { useContext } from 'react';
+
+import { Alert, Skeleton, Typography } from '@mui/material';
+
+import { ItemType, PermissionLevel } from '@graasp/sdk';
+import { NavigationElement, RowMenuProps, RowMenus } from '@graasp/ui';
+
+import { useLibraryTranslation } from '../../../config/i18n';
+import LIBRARY from '../../../langs/constants';
+import { QueryClientContext } from '../../QueryClientContext';
+
+interface RootNavigationTreeProps {
+  isDisabled?: RowMenuProps['isDisabled'];
+  onClick: RowMenuProps['onClick'];
+  onNavigate: RowMenuProps['onNavigate'];
+  rootMenuItems: NavigationElement[];
+  selectedId?: string;
+}
+
+const RootNavigationTree = ({
+  isDisabled,
+  onClick,
+  onNavigate,
+  rootMenuItems,
+  selectedId,
+}: RootNavigationTreeProps): JSX.Element => {
+  const { t } = useLibraryTranslation();
+
+  const { hooks } = useContext(QueryClientContext);
+  // todo: to change with real recent items (most used)
+  const { data: recentItems, isLoading } = hooks.useAccessibleItems(
+    // you can move into an item you have at least write permission
+    {
+      permissions: [PermissionLevel.Admin, PermissionLevel.Write],
+      types: [ItemType.FOLDER],
+    },
+    { pageSize: 5 },
+  );
+  const recentFolders = recentItems?.data?.filter(
+    ({ type }) => type === ItemType.FOLDER,
```

Comment:
As recent Items should be a folder type, Why are we filtering it again?

Reply:
I agree, I think this filter can be removed.